frontends/p4/p4-parse.ypp

```diff
@@ -813,6 +821,24 @@ actionRef
                                            $$ = new IR::ActionListElement(@2, $1, mce); }
     ;
 
+entry
+    : optAnnotations keysetExpression ':' actionRef
+                                         { if($2->is<IR::ListExpression>())
```

Comment:
I actually prefer to convert the expressions to list expressions, so that I can write both:
```
 0xff : actionRef(5);
(0xff, 0x0f) : actionRef(10);
```
this translates to defaults written like this:
```
_ : actionRef(5);
(_, 0x0f) : actionRef(10);
```